web/ci.sh

```diff
@@ -83,4 +90,121 @@ if builder_start_action validate-size; then
   ./src/tools/building/check-build-size.sh $FLAGS
 
   builder_finish_action success validate-size
+fi
+
+if builder_start_action publish-s.keyman; then
+  # First phase: make sure the s.keyman.com repo is locally-available and up to date.
+  pushd "$S_KEYMAN_COM"
+  if builder_has_option --password; then
+    git pull https://keyman-server:$PASSWORD@github.com/keymanapp/s.keyman.com.git master
```

Comment:
I thought keyman-server already has credentials to GitHub. Other CI scripts don't seem to involve $PASSWORD.

Maybe see how help-keyman-com.sh is handling `commit_and_push`

Reply:
The keyboards repo also syncs to s.keyman.com with keyboards/resources/s-keyman-com.sh. Maybe can reuse some of that?

Reply:
Re: the first comment in this chain - note that everything in this script is a near-perfect analogue to Web's existing CI PowerShell scripts... though here, I avoided directly referencing the environment-variable name in favor of having it specified to the script as a parameter.